pybars/_compiler.py

```diff
@@ -898,7 +898,7 @@ def _render(context, helpers=None, partials=None, root=None):
                 'partials': partials,
                 'root': root
             }
-            exec(code + '\nresult = render(context, helpers=helpers, partials=partials, root=root)', ns)
+            exec(code + '\nresult = render(context, helpers=helpers, partials=partials, root=root)') in ns
```

Comment:
It looks like this is now failing in versions 3+ (https://travis-ci.org/wbond/pybars3/builds/550301162?utm_source=github_status&utm_medium=notification)